src/security/expect-ct.conf

```diff
@@ -0,0 +1,28 @@
+# ----------------------------------------------------------------------
+# | Expect-CT                                                          |
+# ----------------------------------------------------------------------
+
+# Opt-in to Certificate Transparency (CT).
+#
+# Certificate Transparency helps guard against several types of certificate-
+# based threats, including misissued certificates, maliciously acquired
+# certificates, and rogue CAs (Certificate Authorities).
+#
+# (!) When a site requires Certificate Transparency (using the `enforce`
+#     directive), they are requesting that the browser check that any
+#     certificate for that site appears in public CT logs (1).
+#
+# It is advised (2) to only serve the Expect-CT header over a secure
+# connection.
+#
+# (1) https://www.certificate-transparency.org/known-logs
```

Comment:
\+ https://crt.sh/ ?

Reply:
&plus; https://www.ssllabs.com/ssltest/?

Reply:
? SSLlabs is not a CT logs explorer, is it?

Reply:
It does provide information about certificate(s), including whether they're trusted or not, here's a screenshot from analyzing https://web.dev:

![ssllabs](https://user-images.githubusercontent.com/26493779/60771345-f7185500-a0e6-11e9-99c7-c6ec7963701e.png)

Reply:
https://crt.sh/ provides the complete log for a single certificate ([ex for web.dev](https://crt.sh/?id=1634127642)).
https://www.ssllabs.com/ssltest/ seems more generic, might be added in the SSL/TLS section but I'm not sure for here.

Reply:
Resolved in https://github.com/h5bp/server-configs-apache/pull/195/commits/9dafb7bdc9fb6502910eb19403c71884e9b650d8.